CODEOWNERS

```diff
@@ -1,3 +1,5 @@
+# cSpell:ignore amtoussam, jlongshore
```

Comment:
So I know I suggested this originally but now that I’m looking at it again, I wonder if `disable-next-line` or `disable-line` would make more sense so that if Security team codeowners ever need to change, we only need to modify one line instead of in two places.